projects/env-semconv.md

```diff
@@ -0,0 +1,69 @@
+### Description
+
+This working group looks to identify and standardize observability for
+sustainability purposes with regards to infrastructure and applications. This
+would look at current tooling around sustainability metrics and standardize the
+meaning and approach for these metrics.
+
+Current efforts of implementations:
+
+- [Kepler](https://github.com/sustainable-computing-io/kepler)
+- [Scaphandre](https://github.com/hubblo-org/scaphandre)
+
+We want to have a unified conventions in order to better facilitate measuring sustainability metrics across different ecosystems.
+
+### Deliverables
+
+- Semantic conventions for sustainability metrics
+- Reference Prototypes metrics
+- Recommended practices for reporting on sustainability metrics within the scope of OpenTelemetry 
+
+### Staffing / Help Wanted
+
+**Project Leads**
+- @spazzy757
+
+**Sponsoring Members**
+- @jpkrohling 
```

Comment:
I can take this @jpkrohling (there's a requirement for sponsors to be from separate companies).